vpr/src/server/gateio.h

```diff
@@ -26,16 +26,16 @@ namespace server {
  * 
  * Operable only with a single client. As soon as client connection is detected
  * it begins listening on the specified port number for incoming client requests,
- * collects and encapsulates them into tasks.
- * The incoming tasks are extracted and handled by the top-level logic (TaskResolver). 
- * Once the tasks are resolved by the TaskResolver, they are returned 
- * to be sent back to the client as a response.
+ * collects and encapsulates them into tasks (see @ref Task).
+ * The incoming tasks are extracted and handled by the top-level logic @ref TaskResolver in main thread.
+ * Once the tasks are resolved by the @ref TaskResolver, they are returned to be sent back to the client app as a response.
+ * Extraction and puting @ref Task is taken from main thread inside @ref server::update.
```

Comment:
puting (putting?) -- not clear what this means.
from main thread -> from the main thread

Reply:
solved https://github.com/w0lek/vtr-verilog-to-routing/pull/11/commits/c92e52cbd32fb2d21290d0e59f6bac027373a589
(will be updated later)